src/App.js

```diff
@@ -1,23 +1,61 @@
-import logo from './logo.svg';
+
+
+
+import {useState, useEffect} from "react";
```

Comment:
What do you think about my proposal to move the starting line of code to line 1 of app.js?